### ISSUE TITLE
Several links not working (https://albumentations.ai/docs/autoalbument/custom_model/)

### ISSUE THREAD

**Issue by andife** [albumentations-team/autoalbument#24]
The following links are broken:

error Line: 1980 https://hydra.cc/docs/next/patterns/instantiate_objects/overview/
Status: 404 Not Found
The link is broken. Double-check that you have not made any typo, or mistake in copy-pasting. If the link points to a resource that no longer exists, you may want to remove or fix the link.

error Line: 40 https://fonts.gstatic.com/
Status: 404 Not Found
The link is broken. Double-check that you have not made any typo, or mistake in copy-pasting. If the link points to a resource that no longer exists, you may want to remove or fix the link.

error Line: 47 https://albumentations.ai/docs/assets/_mkdocstrings.css
Status: 404 Not Found
The link is broken. Double-check that you have not made any typo, or mistake in copy-pasting. If the link points to a resource that no longer exists, you may want to remove or fix the link.

error Lines: 1977, 1978 https://github.com/albumentations-team/autoalbument/blob/master/autoalbument/faster_autoaugment/models.py

You can see the results also here:
https://validator.w3.org/checklink?uri=https%3A%2F%2Falbumentations.ai%2Fdocs%2Fautoalbument%2Fcustom_model%2F&hide_type=all&recursive=on&depth=2&check=Check

**Comment by ihamdi** [comment]
Yeah the only example I managed to get working is cifar10. Everything else is broken.
